build.gradle

```diff
@@ -88,7 +89,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "512m"
+    maxHeapSize = "1024m"
```

Comment:
Now that you have cleaned up associations that were made in the different stores, I would be interested in what happens if you revert back to 512m